index.js

```diff
@@ -45,40 +45,50 @@ var cartography = (function() {
       if (typeof args[0] === 'string') {
         var sourceAttributePath = args[0];
         var filters = args.slice(1);
-        var value = sourceAttributePath.split('.').reduce(function(o, step) {return (o || {})[step];}, source);
+        var value = sourceAttributePath.split('.').reduce(function (o, step) {
+          return (o || {})[step];
+        }, source);
       }
       // All arguments are filter functions
-      else
-      {
+      else {
         var sourceAttributePath = destinationAttribute;
         var filters = args;
         var value = (source || {})[sourceAttributePath];
       }
 
-      function formatError (message) {return addSeparator(sourceAttributePath, message);}
+      function formatError(message) {
+        return addSeparator(sourceAttributePath, message);
+      }
       return runFiltersOn(value, filters, formatError);
     }
 
     // Build destination attribute via a custom function
-    function callCustomFunction(fn) {return fn(source);}
+    function callCustomFunction(fn) {
+      return fn(source);
+    }
 
     // Destination attribute is an Object with a nested schema
-    function goRecursive(schema) {return map(source, schema);}
+    function goRecursive(schema) {
+      return map(source, schema);
+    }
 
     // Error: a string was used. Throw an exception
-    function rejectString(s, destinationAttribute) {throw new Error('invalid schema for `'+destinationAttribute+'`');}
+    function rejectString(s, destinationAttribute) {
+      throw new Error('invalid schema for `' + destinationAttribute + '`');
+    }
 
     var destination = {};
     var length = 0;
     for (var destinationAttribute in schema) {
-
       var value = schema[destinationAttribute];
 
-      var method =
-        Array.isArray(value) ? runFilters :
-        typeof value === 'function' ? callCustomFunction :
-        typeof value === 'string' ? rejectString :
-        goRecursive;
+      var method = Array.isArray(value)
+        ? runFilters
```

Comment:
this cause the `laxbreak` rule to fail. I will do a followup PR to use eslint instead of jshint